src/Controller/Admin/UserCrudController.php

```diff
@@ -22,7 +30,8 @@ class UserCrudController extends AbstractCrudController
 {
     public function __construct(
         private UserPasswordHasherInterface $hasher,
-    ) {}
+    ) {
+    }
```

Comment:
Any coding guideline that requires this much pointless extra typing is wrong and should be fixed.  Yes, that includes PSr-12, which is wrong.

Reply:
What ever you like, just adapt it and make it auto fixable